code/modules/mob/mob.dm

```diff
@@ -1073,12 +1073,14 @@ var/list/slot_equipment_priority = list( \
 
 /mob/verb/cancel_camera()
 	set name = "Cancel Camera View"
-	set category = "OOC"
-	reset_view(null)
+	set category = "OOC" //Why the fuck?
 	unset_machine()
+	reset_view(null)
 	if(istype(src, /mob/living))
 		if(src:cameraFollow)
 			src:cameraFollow = null
+		if(istype(src, /mob/living/carbon/human))
+			src:handle_regular_hud_updates()
```

Comment:
Is that slower or something? It seems like it would use more overhead to define a variable and then use a dot than it would to do it this way, but I don't really know.